ThingIFSDK/ThingIFSDKLargeTests/LargeTestBase.swift

```diff
@@ -143,71 +138,71 @@ class LargeTestBase: XCTestCase {
-        request.addValue(
+            hostName: String) -> Dictionary<String, AnyObject>
+    {
+        var request = URLRequest(
+          url: URL(string: "https://\(hostName)/api/apps/\(appID)/users")!)
+        request.httpMethod = "POST"
+        request.allHTTPHeaderFields = [
+          "X-Kii-AppID" : appID,
+          "X-Kii-AppKey" : appKey,
+          "Content-Type" :
             "application/vnd.kii.RegistrationAndAuthorizationRequest+json",
-            forHTTPHeaderField: "Content-Type")
-        request.HTTPBody =
-            ("{}" as NSString).dataUsingEncoding(NSUTF8StringEncoding)
+        ]
+        request.httpBody =
+            ("{}" as NSString).data(using: String.Encoding.utf8.rawValue)
 
-        let expectation = self.expectationWithDescription("Create user")
+        let expectation = self.expectation(description: "Create user")
 
         let session =
-            NSURLSession(configuration: NSURLSessionConfiguration.defaultSessionConfiguration())
-        var data: NSData?
-        let dataTask = session.dataTaskWithRequest(
-                           request,
-                           completionHandler: { (receivedData, response, error) -> Void in
-                               data = receivedData
-                               expectation.fulfill()
-            })
+            URLSession(configuration: URLSessionConfiguration.default)
+        var data: Data?
+        let dataTask = session.dataTask(with: request) {
+            receivedData, response, error -> Void in
+            data = receivedData
+            expectation.fulfill()
+        }
         dataTask.resume()
-        self.waitForExpectationsWithTimeout(TEST_TIMEOUT) { (error) -> Void in
+        self.waitForExpectations(timeout: TEST_TIMEOUT) { (error) -> Void in
             if error != nil {
                 XCTFail("error")
             }
         }
 
-        return try! NSJSONSerialization.JSONObjectWithData(
-                   data!,
-                   options:.AllowFragments) as! Dictionary<String, AnyObject>
+        return try! JSONSerialization.jsonObject(
+                   with: data!,
+                   options:.allowFragments) as! Dictionary<String, AnyObject>
     }
 
     func deletePseudoUser(
-            appID: String,
+            _ appID: String,
             appKey: String,
             userID: String,
             accessToken: String,
-            hostName: String) -> Void {
-        let request = NSMutableURLRequest(
-                URL: NSURL(string: "https://\(hostName)/api/apps/\(appID)/users/\(userID)")!)
-        request.HTTPMethod = "DELETE"
-        request.addValue(appID, forHTTPHeaderField: "X-Kii-AppID")
-        request.addValue(appKey, forHTTPHeaderField: "X-Kii-AppKey")
-        request.addValue("Bearer \(accessToken)", forHTTPHeaderField: "Authorization")
-
-        let expectation = self.expectationWithDescription("Delete user")
+            hostName: String) -> Void
```

Comment:
better remove `-> Void`

reff : https://developer.apple.com/reference/swift/void
```
When declaring a function or method, you don’t need to specify a return type if no value will be returned. However, the type of a function, method, or closure always includes a return type, which is Void if otherwise unspecified.
```